src/data/chats/chat_entity.rs

```diff
@@ -0,0 +1,38 @@
+//! Hopefully, provisional solution to unify model files and agents in the chat system.
```

Comment:
Just moved these glues to a separate file hoping we find a way to really unify models and agents later.